src/commands/daily.ts

```
@@ -167,3 +290,835 @@ export const dailyCommand = define({
+			// Test the groupDataByProject function
+			const projectGroups = groupDataByProject(mockDailyData);
+
+			// Should have 2 project groups
+			expect(Object.keys(projectGroups)).toHaveLength(2);
+			expect(projectGroups['project-a']).toBeDefined();
+			expect(projectGroups['project-b']).toBeDefined();
+
+			// Each group should contain correct data
+			expect(projectGroups['project-a']).toHaveLength(1);
+			expect(projectGroups['project-b']).toHaveLength(1);
+			expect(projectGroups['project-a']![0]!.inputTokens).toBe(1000);
```

ESLint

[error] Unsafe call of a(n) `error` type typed value.

(ts/no-unsafe-call)